packages/destination-actions/src/destinations/pipedrive/createUpdateActivity/index.ts

```diff
@@ -0,0 +1,183 @@
+import type { ActionDefinition } from '@segment/actions-core'
+import type { Settings } from '../generated-types'
+import type { Payload } from './generated-types'
+import { Activity, createActivity } from '../pipedriveApi/activities'
+import PipedriveClient from '../pipedriveApi/pipedrive-client'
+
+const action: ActionDefinition<Settings, Payload> = {
+  title: 'Create or Update an Activity',
+  description: "Update an Activity in Pipedrive or create it if it doesn't exist yet.",
+  defaultSubscription: 'type = "track"',
+  fields: {
+    person_match_field: {
+      label: 'Person match field',
+      description: 'If present, used instead of field in settings to find existing person in Pipedrive.',
+      type: 'string',
+      required: false,
+      dynamic: true,
+      default: {
+        '@literal': 'id'
+      },
```

Comment:
```suggestion
      default: 'id',
```

Literal values can be assigned directly to defaults. No need for the `@literal` directive (though that works fine, too).